invenio_rdm_migrator/streams/models/oauth.py

```diff
@@ -45,15 +45,15 @@ class ServerClient(Model):
     __tablename__: InitVar[str] = "oauth2server_client"
 
     name: Mapped[str]
-    description: Mapped[str]
-    website: Mapped[str]
-    user_id: Mapped[int]
     client_id: Mapped[str] = mapped_column(primary_key=True)
+    description: Mapped[str] = mapped_column(nullable=True)
+    website: Mapped[str] = mapped_column(nullable=True)
+    user_id: Mapped[int] = mapped_column(nullable=True)
     client_secret: Mapped[str]  # it is text, migrated as is
-    is_confidential: Mapped[bool]
-    is_internal: Mapped[bool]
-    _redirect_uris: Mapped[str]
-    _default_scopes: Mapped[str]
+    _redirect_uris: Mapped[str] = mapped_column(nullable=True)
```

Comment:
all columns, unless specified the contrary in the invenio models  are nullable https://github.com/sqlalchemy/sqlalchemy/blob/rel_1_3_24/lib/sqlalchemy/sql/schema.py#L1509 in sqlalchemy 1.3.24